README.md

```diff
@@ -10,7 +10,7 @@ This project enables Java code generation in [AutoRest](https://github.com/Azure
   - Default: `true`
   - Without Argument: `int Plus(Integer val1, Integer val2)`
   - With Argument: `int Plus(@NonNull Integer val1, @NonNull Integer val2)`
-- *--java.package-private*: Whether or not all created types will be package-private.
+- *--java.package-private*: Whether or not created client types (ServiceClient and MethodGroupClients) will be package-private.
```

Comment:
A package-private type can only be referenced by a type in the same package. Unfortunately something like accessing the type from a subpackage doesn't work. Since e.g. `MyMethodGroupImpl implements MyMethodGroup`, code generated with this flag won't compile.